SportsGoods.Api/SportsGoods.Api.csproj

```diff
@@ -9,6 +9,11 @@
   <ItemGroup>
     <PackageReference Include="MediatR" Version="12.2.0" />
     <PackageReference Include="Microsoft.EntityFrameworkCore" Version="8.0.3" />
```

Comment:
No unit testing stuff in this project @LVarsaminov

Reply:
Done (removed)